packages/deno/src/integrations/vercelai.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * This is a copy of the Vercel AI integration from the cloudflare SDK.
+ *
+ * The only difference is that it does not use `@opentelemetry/instrumentation`
+ * because Deno Workers do not support it in the same way.
+ *
+ * Therefore, we cannot automatically patch setting `experimental_telemetry: { isEnabled: true }`
+ * and users have to manually set this to get spans.
```

Comment:
I think this comment is wrong, I would just remove it - this is just a 1-1 copy of the cloudflare one, right? :D

So either we just say "this is a copt of the cloudflare one" or we copy the whole comment and say "it is a copy of the node one, but...."